.buildkite/smart_exhaustive_tests_pipeline.yml

```diff
@@ -0,0 +1,33 @@
+steps:
```

Comment:
As long as the BK pr bot does not support running the very first build for PRs. We use this intermediate pipeline. This will help with running faster builds regardless, although it adds the complexity for maintaining a new pipeline.